replica/database.hh

```diff
@@ -409,6 +410,7 @@ private:
     lw_shared_ptr<memtable_list> make_memory_only_memtable_list();
     lw_shared_ptr<memtable_list> make_memtable_list();
 
+    compaction_manager& _compaction_manager;
```

Comment:
nit: why moving _compaction_manager here is needed?

Reply:
That's because table::_compaction_manager should be set before group accesses it.

Reply:
ok